stingray/mission_support/rxte.py

```diff
@@ -0,0 +1,337 @@
+        two arguments: the PHA channel and the PCU number.
+
+    Examples
+    --------
+    >>> conversion_function = pca_calibration_func(50082)
+    >>> float(conversion_function(10, 0))
+    3.04
+    >>> conversion_function = pca_calibration_func(55930)
+    >>> float(conversion_function(10, 0))
+    4.53
+    >>> float(conversion_function(10, 3))
+    4.49
+    >>> assert np.array_equal(conversion_function(10, [0, 3]), [4.53, 4.49])
+    >>> assert np.array_equal(conversion_function([10, 11], [0, 3]), [4.53, 4.90])
+    """
+    caltable = Table.read(
+        """
+    Abs   STD2  E1   E2    E3   E4   E5_0   E5_1234
+    0-4	0	1.51	1.61	1.94	2.13	1.95	  2.06
+    5	1	1.76	1.91	2.29	2.54	2.38	  2.47
+    6	2	2.02	2.22	2.64	2.96	2.81	  2.87
+    7	3	2.27	2.52	2.99	3.37	3.24	  3.28
+    8	4	2.53	2.83	3.35	3.79	3.67	  3.68
+    9	5	2.78	3.13	3.70	4.21	4.09	  4.09
+    10	6	3.04	3.44	4.05	4.63	4.53	  4.49
+    11	7	3.30	3.75	4.41	5.04	4.96	  4.90
+    12	8	3.55	4.05	4.76	5.46	5.39	  5.31
+    13	9	3.81	4.36	5.12	5.88	5.82	  5.71
+    14	10	4.07	4.67	5.47	6.30	6.25	  6.12
+    15	11	4.32	4.98	5.82	6.72	6.68	  6.53
+    16	12	4.58	5.28	6.18	7.14	7.11	  6.94
+    17	13	4.84	5.59	6.54	7.56	7.55	  7.35
+    18	14	5.10	5.90	6.89	7.98	7.98	  7.76
+    19	15	5.35	6.21	7.25	8.40	8.42	  8.17
+    20	16	5.61	6.52	7.60	8.82	8.85	  8.57
+    21	17	5.87	6.83	7.96	9.24	9.28	  8.98
+    22	18	6.13	7.14	8.32	9.67	9.72	  9.40
+    23	19	6.39	7.45	8.68	10.09	10.16	  9.81
+    24	20	6.65	7.76	9.03	10.51	10.59	  10.22
+    25	21	6.91	8.07	9.39	10.93	11.03	  10.63
+    26	22	7.17	8.38	9.75	11.36	11.47	  11.04
+    27	23	7.43	8.69	10.11	11.78	11.90	  11.45
+    28	24	7.69	9.00	10.47	12.21	12.34	  11.87
+    29	25	7.95	9.31	10.83	12.63	12.78	  12.28
+    30	26	8.21	9.63	11.19	13.06	13.22	  12.69
+    31	27	8.47	9.94	11.55	13.48	13.66	  13.11
+    32	28	8.73	10.25	11.91	13.91	14.10	  13.52
+    33	29	8.99	10.56	12.27	14.34	14.54	  13.93
+    34	30	9.25	10.88	12.63	14.76	14.98	  14.35
+    35	31	9.52	11.19	12.99	15.19	15.42	  14.76
+    36	32	9.78	11.50	13.36	15.62	15.86	  15.18
+    37	33	10.04	11.82	13.72	16.05	16.30	  15.60
+    38	34	10.30	12.13	14.08	16.47	16.74	  16.01
+    39	35	10.57	12.45	14.44	16.90	17.19	  16.43
+    40	36	10.83	12.76	14.81	17.33	17.63	  16.85
+    41	37	11.09	13.08	15.17	17.76	18.07	  17.26
+    42	38	11.36	13.39	15.54	18.19	18.52	  17.68
+    43	39	11.62	13.71	15.90	18.62	18.96	  18.10
+    44	40	11.89	14.03	16.26	19.05	19.41	  18.52
+    45	41	12.15	14.34	16.63	19.49	19.85	  18.94
+    46	42	12.42	14.66	17.00	19.92	20.30	  19.36
+    47	43	12.68	14.98	17.36	20.35	20.75	  19.78
+    48	44	12.95	15.29	17.73	20.78	21.19	  20.20
+    49	45	13.21	15.61	18.09	21.22	21.64	  20.62
+    50	46	13.48	15.93	18.46	21.65	22.09	  21.04
+    51	47	13.74	16.25	18.83	22.08	22.54	  21.46
+    52	48	14.01	16.57	19.20	22.52	22.98	  21.88
+    53	49	14.28	16.89	19.56	22.95	23.43	  22.30
```

Comment:
The format of the table changes here, it can get confusion to read and comparte

Reply:
This is the original table from the RXTE website, I maintained their format. I preferred to make the original table available instead of storing in a different format. Probably an excess of caution, but I figured it would have been easier to track possible changes over time

Reply:
I remember that format from years ago...I understand your point, even though looking at the table formatted in that way make me think that many mistakes had been and will be made :)

Reply:
I feel the pain